repo/meta/index.json

```diff
@@ -95,6 +95,7 @@
       ],
       "versions":{
         "0.8.1":"0",
+        "0.9.0":"2",
         "0.9.0-RC3":"1"
```

Comment:
Yes. Just an aside we have standardize on package version to mean the software version. E.g. Marathon 0.8.1, 0.9.0, etc. And package revision to mean 0 vs 1 vs 2. We need to change the Universe documentation to reflect that.